types/filesize.d.ts

```diff
@@ -1,32 +1,63 @@
-export interface FileSizeOptions {
+interface FileSizeOptionsBase {
+    base?: number;
     bits?: boolean;
+    exponent?: number;
+    fullform?: boolean;
+    fullforms?: string[];
+    locale?: string | boolean;
+    localeOptions?: Intl.DateTimeFormatOptions;
     pad?: boolean;
-    base?: number;
+    precision?: number;
     round?: number;
-    locale?: string;
-    localeOptions?: {};
+    roundingMethod?: 'round' | 'floor' | 'ceil';
     separator?: string;
     spacer?: string;
-    symbols?: {};
     standard?: 'iec' | 'jedec';
-    output?: 'array' | 'exponent' | 'object' | 'string';
```

Comment:
This is very important to not change the behaviour of.

Reply:
You are correct. This was also not changed, but rather I overlooked the case that if other options are given and NOT `output` it also should return string and not force the setting of `output`. I will wait on the issue opener to create a PR, otherwise I can if wanted.